tests/test_wave_03.py

```diff
@@ -1,212 +1,77 @@
-                        "title": "Title A"
-                    },
-                    {
-                        "title": "Title C"
-                    }
-                ]
-            },
-            {
-                "watched": [
-                    {
-                        "title": "Title A"
-                    },
-                    {
-                        "title": "Title D"
-                    },
-                    {
-                        "title": "Title F"
-                    }
-                ]
-            }
-        ]
-    }
+    amandas_data = USER_DATA_3
 
     # Act
     amandas_unique_movies = get_unique_watched(amandas_data)
 
     # Arrange
     assert len(amandas_unique_movies) is 2
-    assert {"title": "Title B"} in amandas_unique_movies
-    assert {"title": "Title E"} in amandas_unique_movies
-
+    assert FANTASY_2 in amandas_unique_movies
+    assert INTRIGUE_2 in amandas_unique_movies
+    assert amandas_data is USER_DATA_3
 
+@pytest.mark.skip()
 def test_my_not_unique_movies():
     # Arrange
-    amandas_data = {
-        "watched": [],
-        "friends": [
-            {
-                "watched": [
-                    {
-                        "title": "Title A"
-                    },
-                    {
-                        "title": "Title C"
-                    }
-                ]
-            },
-            {
-                "watched": [
-                    {
-                        "title": "Title A"
-                    },
-                    {
-                        "title": "Title D"
-                    },
-                    {
-                        "title": "Title F"
-                    }
-                ]
-            }
-        ]
-    }
+    amandas_data = copy.deepcopy(USER_DATA_3)
+    amandas_data["watched"] = []
 
     # Act
     amandas_unique_movies = get_unique_watched(amandas_data)
 
     # Arrange
     assert len(amandas_unique_movies) is 0
 
-
+@pytest.mark.skip()
 def test_friends_unique_movies():
     # Arrange
-    amandas_data = {
-        "watched": [
-            {
-                "title": "Title A"
-            },
-            {
-                "title": "Title B"
-            },
-            {
-                "title": "Title C"
-            }
-        ],
-        "friends": [
-            {
-                "watched": [
-                    {
-                        "title": "Title A"
-                    },
-                    {
-                        "title": "Title C"
-                    }
-                ]
-            },
-            {
-                "watched": [
-                    {
-                        "title": "Title A"
-                    },
-                    {
-                        "title": "Title D"
-                    },
-                    {
-                        "title": "Title E"
-                    }
-                ]
-            }
-        ]
-    }
+    amandas_data = USER_DATA_3
```

Comment:
something something pytest fixtures instead of using mutable global variables? (sry, still a a python n00b ;__; )

Reply:
update: this is likely happening because my implementation of `get_unique_watched` mutates the `user_data` 😬

Reply:
per our conversation, we don't want students to mutate `user_data`, but it's also a problem that our unit tests don't start fresh data each time. I will think more on how to handle this. Introducing fixtures now feels too early, so it may mean copying and pasting the data into each test.

Reply:
The approach of pulling the data out into a `test_constants.py` file came from writing integration tests. We decided to scrap the integration tests because they were really unit tests stuck together, but I think this same approach doesn't work well for unit tests.